examples/src/main/java/com/mapbox/navigation/examples/core/MapboxVoiceActivity.kt

```diff
@@ -270,7 +259,26 @@ class MapboxVoiceActivity : AppCompatActivity(), OnMapLongClickListener {
         soundButtonMake(isMuted) // init state
 
         binding.soundButton.setOnClickListener {
-            soundButtonMake(!isMuted)
+            // soundButtonMake(!isMuted)
+            voiceInstructionsCache.forEach {
+                speechApi.generate(it) { expected ->
+                    expected.fold(
+                        { error ->
+                            Log.e("DEBUG", "DEBUG error: ${error.errorMessage}")
+                        },
+                        { value ->
+                            Log.e(
+                                "DEBUG",
+                                "DEBUG ssmlAnnouncement: ${value.announcement.ssmlAnnouncement}"
+                            )
+                            Log.e(
+                                "DEBUG",
+                                "DEBUG ssmlAnnouncement file: ${value.announcement.file}"
+                            )
+                        }
+                    )
+                }
+            }
```

Comment:
Good to revert?

Reply:
Yeah I was on it while fixing the conflict with `main` 😅